relay-server/src/actors/spooler/mod.rs

```diff
@@ -396,10 +425,19 @@ impl OnDisk {
             .try_get("received_at")
             .map_err(BufferError::FetchFailed)?;
         let start_time = StartTime::from_timestamp_millis(received_at as u64);
```

Comment:
isnt it better to fetch received_at as u64 from the DB, i guess that would return an error rather than converting to huge number if negative?

Reply:
I assume this is because SQLite does not know the `u64` datatype only `i64`

Reply:
this is not supported by sqlx, only `i64` in this case